File: app/src/main/java/com/novel/read/event/HotSearchEvent.java
package com.novel.read.event;

import com.mango.mangolib.event.BaseEvent;
import com.novel.read.model.protocol.HotSearchResp;

/**
 * create by 赵利君 on 2019/6/18
 * describe:
 */
public class HotSearchEvent extends BaseEvent<HotSearchResp> {

    public HotSearchEvent(HotSearchResp result) {
        super(result);
    }

    public HotSearchEvent() {
    }
}

File: app/src/main/java/com/novel/read/event/HotSearchEvent.kt
package com.novel.read.event

import com.mango.mangolib.event.BaseEvent
import com.novel.read.model.protocol.HotSearchResp
import com.novel.read.model.protocol.SearchResp

/**
 * create by zlj on 2019/6/18
 * describe:
 */
class HotSearchEvent : BaseEvent<HotSearchResp>{

    constructor(result: HotSearchResp) : super(result) {}

    constructor() {}
}
